views/index.html.twig

```diff
@@ -1,34 +1,81 @@
 {% extends "layout.html.twig" %}
 
+
 {% block header %}
 {% endblock %}
 
+
 {% block content %}
-    <div class="logo">
-        <img src="{{ app.request.basePath }}/img/{{ logo }}">
-        <h1 class="slogan">
-            Dependency Manager for PHP
-        </h1>
-        <p class="latest">Latest: v{{ latestStable }}</p>
-    </div>
-
-    <p class="buttons">
-        <a href="{{ path('docs.view', {page: '00-intro.md'}) }}">Getting Started</a>
-        <a href="{{ path('download') }}">Download</a>
-    </p>
-    <p class="buttons">
-        <a href="{{ path('docs') }}">Documentation</a>
-        <a href="https://packagist.org/">Browse Packages</a>
-    </p>
-    <p class="buttons">
-        <a href="https://github.com/composer/composer/issues">Issues</a>
-        <a href="https://github.com/composer/composer">GitHub</a>
-    </p>
+
+    <section role="banner">
+        <div class="logo">
+            <figure>
+                <img src="{{ app.request.basePath }}/img/{{ logo }}"
+                     alt="The Composer Official Logo: a male orchestra conductor with both arms in the air and his head tilted down, reading music sheets"/>
+            </figure>
+            <h1 class="slogan" role="definition">
+                A Dependency Manager for PHP
+            </h1>
+            <p class="latest">Latest: <strong>{{ latestStable }}</strong></p>
+        </div>
+    </section>
+
+    <article role="article">
+        <p class="buttons">
+            <a href="{{ path('docs.view', {page: '00-intro.md'}) }}"
+               title="Getting started with Composer" aria-label="Getting started with Composer">Getting Started</a>
+            <a href="{{ path('download') }}"
+               title="How to download Composer" aria-label="How to download Composer">Download</a>
+        </p>
+        <p class="buttons">
+            <a href="{{ path('docs') }}"
+               title="View the Composer documentation"
+               aria-label="View the Composer documentation">Documentation</a>
+            <a href="https://packagist.org/"
+               title="Browse Composer packages on packagist.org (external link)"
+               aria-label="Browse Composer packages on packagist.org (external link)">Browse Packages</a>
+        </p>
+        <p class="buttons">
+            <a href="https://github.com/composer/composer/issues"
+               title="View current issues on GitHub (external link)"
```

Comment:
Well, my understanding is that: 
- alt is useful for alternatives because of loading, and also for referecing
- title is useful for all browsers, and offers a visual description, bubbled and / or read, depending on browsers
- ARIA attributes offer further alternates for browsers with disability compensations.
- link (or any other markup) content is made to be inserted in the flow, sometimes depending upon contextual grammar, or presentation

So, in short, you set...
- link content to reflect what you want to say in context, neutrally
- title to explain what the link does to anyone (like describing if the link goes to an external source, for example)
- aria-label for the specific disabilities (like describing what the link looks like, for example)

Also, looking at the compatibility charts, it seems ARIA is better supported than title, and whatever, they don't have the same support. So using both can never be a bad idea.

This answer should help: https://stackoverflow.com/a/32560017

Reply:
I'll close that, too, unless you really feel that it's a real drawback instead of a progress, @stof. I'm pretty sure this goes forward. Feel free to reopen.